README.md

```diff
@@ -40,3 +40,6 @@ Installing cli-tools automatically fetches additional dependencies as needed.
 
 Further documentation: https://github.com/iot-lab/iot-lab/wiki/CLI-Tools
 
+
+[build-icon]: https://travis-ci.org/iot-lab/cli-tools.svg?branch=master
```

Comment:
I prefer doing this in an other PR with full tox unittests integration (similar to ssh-cli-tools).